solutions/devsprint-bruno-ramos-2/DeliveryAppChallenge/Modules/Product/Home/HomePresenter.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  HomePresenter.swift
+//  DeliveryAppChallenge
+//
+//  Created by Leonardo Mendes on 30/01/22.
+//
+
+import Foundation
+
+/// Protocol used to intermediate communication from Interactor to Presenter ( Interactor ----`response` ----> Presenter )
+protocol HomePresentationLogic {
+    func responseData(response: HomeUseCase.FetchData.Response)
```

Comment:
Nesse caso aqui ao invés de response, seria present, então podemos mudar o nome dos dois métodos
```suggestion
    func presentFetchedRestauantList(response: HomeUseCase.FetchData.Response)
```

Reply:
Corrigido

Reply:
Ah sim, sem problemas. Nesse caso acho que a gente pode deixar mais exemplos nele de nomenclaturas, nesse caso como era uma visão mais macro, mantive só os parâmetros normalmente que são passados. My bad :(

Reply:
nao poh, que isso
so to explicando o meu pensamento para colocar os nomes msm